sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Functions.scala

```diff
@@ -1287,3 +1287,16 @@ case class ST_Normalize(inputExpressions: Seq[Expression])
     copy(inputExpressions = newChildren)
   }
 }
+
+/**
+ * Returns the LineString geometry given a MultiPoint geometry
+ *
+ * @param inputExpressions
```

Comment:
Please add a DataFrame style API in Python and Scala for this function. See PR #693 .